lib/rouge/lexers/graphql.rb

```diff
@@ -20,19 +20,25 @@ class GraphQL < RegexLexer
         end
 
         rule %r/\bfragment\b/, Keyword, :fragment_definition
+        rule %r/\bscalar\b/, Keyword, :value
+        rule %r/\bextend\sscalar\b/, Keyword, :value
```

Comment:
Since the `extend` keyword can be used before multiple keywords, I'd suggest a better way to do what you're trying to do here is to have a rule that matches `\bextend\b` and sets it to have the `Keyword` token (but doesn't pass a new state).

This will mean you can use `extend` in ways that are syntax errors in GraphQL but that's fine. This is merely a lexer, not a parser, and checking syntax correctness shouldn't be a concern.